docs/ref/django-admin.txt

```diff
@@ -222,6 +222,10 @@ program manually.
 
 Specifies the database onto which to open a shell. Defaults to ``default``.
 
+.. note::
+
+    Some of ``DATABASES['OPTIONS']`` are not passed to the underlying dbshell
```

Comment:
Can you try to be more expressive? I needed to look into the issue to understand what your comment was about. Maybe add an example – like `isolation_level`. Furthermore, I didn't get at first that you are talking about the management command `dbshell`. I was wondering what you mean by underlying shell, since there is no 'shell' under a Django database connection but only a database driver.

Please also make sure to end each sentence on a full stop.